hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/RabinFingerprintTest.java

```diff
@@ -36,6 +36,17 @@
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class RabinFingerprintTest {
 
+    @Test
+    public void testRabinFingerprint() {
```

Comment:
I wonder why you added this test (who offered to add it). I think we needed such a test. I added a similar test in the go client.  Also how did you compute the magic number in this test?

https://github.com/hazelcast/hazelcast-go-client/pull/782/files#diff-9101252afd5300fe01384c3b9e9f676b67e31289728c71f605d685a396ef4093R27

Reply:
I added this test to make sure that I broke nothing in the process of changing these files. It is just a simple sanity check.

I simply calculated the schema id before doing the changes using something similar to what you linked in the commit